{
    "kumparan.py-master/setup.py": {},
    "kumparan.py-master/kumparan/service_account.py": {
        "get_credentials": {
            "name": "get_credentials",
            "location": 12,
            "return": [],
            "arguments": {}
        }
    },
    "kumparan.py-master/kumparan/stackdriver.py": {
        "main": {
            "name": "main",
            "location": 78,
            "return": [],
            "arguments": {}
        },
        "StackdriverLoggingFormatter.__init__": {
            "name": "__init__",
            "location": 27,
            "return": [],
            "arguments": {
                "self": [],
                "service_name": [],
                "service_version": []
            }
        },
        "StackdriverLoggingFormatter.formatTime": {
            "name": "formatTime",
            "location": 40,
            "return": [
                "str"
            ],
            "arguments": {
                "self": [],
                "record": [
                    "logging.LogRecord"
                ],
                "datefmt": [
                    "str"
                ]
            }
        },
        "StackdriverLoggingFormatter.format": {
            "name": "format",
            "location": 52,
            "return": [
                "str"
            ],
            "arguments": {
                "self": [],
                "record": [
                    "logging.LogRecord"
                ]
            }
        }
    },
    "kumparan.py-master/kumparan/utils.py": {
        "unixtimestamp": {
            "name": "unixtimestamp",
            "location": 8,
            "return": [
                "int"
            ],
            "arguments": {}
        }
    },
    "kumparan.py-master/kumparan/__init__.py": {},
    "kumparan.py-master/tests/service_account_test.py": {
        "TestSupervisedAccount.test_get_credentials_not_set": {
            "name": "test_get_credentials_not_set",
            "location": 9,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestSupervisedAccount.test_get_credentials_invalid_json": {
            "name": "test_get_credentials_invalid_json",
            "location": 16,
            "return": [],
            "arguments": {
                "self": []
            }
        }
    },
    "kumparan.py-master/tests/stackdriver_test.py": {
        "init_logger": {
            "name": "init_logger",
            "location": 9,
            "return": [
                "logging.Logger"
            ],
            "arguments": {
                "output_path": [
                    "str"
                ],
                "service_name": [
                    "str"
                ],
                "service_version": [
                    "str"
                ]
            }
        },
        "TestStackdriverLogging.test_severity_debug": {
            "name": "test_severity_debug",
            "location": 39,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestStackdriverLogging.test_severity_info": {
            "name": "test_severity_info",
            "location": 64,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestStackdriverLogging.test_severity_warning": {
            "name": "test_severity_warning",
            "location": 89,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestStackdriverLogging.test_severity_error": {
            "name": "test_severity_error",
            "location": 114,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestStackdriverLogging.test_severity_critical": {
            "name": "test_severity_critical",
            "location": 139,
            "return": [],
            "arguments": {
                "self": []
            }
        },
        "TestStackdriverLogging.test_severity_exception": {
            "name": "test_severity_exception",
            "location": 164,
            "return": [],
            "arguments": {
                "self": []
            }
        }
    },
    "kumparan.py-master/tests/utils_test.py": {
        "TestUtils.test_unixtimestamp": {
            "name": "test_unixtimestamp",
            "location": 7,
            "return": [],
            "arguments": {
                "self": []
            }
        }
    }
}